extensions/smallrye-openapi-common/deployment/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -1,12 +0,0 @@
----
-name: "SmallRye OpenAPI Common"
-metadata:
-  keywords:
-  - "smallrye-openapi"
-  - "openapi"
-  - "open-api"
-  guide: "https://quarkus.io/guides/openapi-swaggerui"
-  categories:
-  - "web"
-  status: "stable"
-  unlisted: true
```

Comment:
Why did you delete this file?

Reply:
This file does not belong in the deployment artifact. And there is no corresponding runtime one.